alfresco-docker-tika/src/main/resources/engine_config.json

```diff
@@ -12,14 +12,292 @@
+        {"sourceMediaType": "application/vnd.ms-excel.sheet.binary.macroenabled.12",                       "targetMediaType": "text/plain"},
+        {"sourceMediaType": "application/vnd.ms-excel.sheet.binary.macroenabled.12",                       "targetMediaType": "application/xhtml+xml"},
+        {"sourceMediaType": "application/vnd.ms-excel.sheet.binary.macroenabled.12",                       "targetMediaType": "text/xml"},
+
+        {"sourceMediaType": "application/vnd.ms-excel.sheet.macroenabled.12",                              "targetMediaType": "text/html"},
+        {"sourceMediaType": "application/vnd.ms-excel.sheet.macroenabled.12",                              "targetMediaType": "text/plain"},
+        {"sourceMediaType": "application/vnd.ms-excel.sheet.macroenabled.12",                              "targetMediaType": "application/xhtml+xml"},
+        {"sourceMediaType": "application/vnd.ms-excel.sheet.macroenabled.12",                              "targetMediaType": "text/xml"},
+
+        {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",           "targetMediaType": "text/html"},
+        {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",           "targetMediaType": "text/plain"},
+        {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",           "targetMediaType": "application/xhtml+xml"},
+        {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",           "targetMediaType": "text/xml"},
+
+        {"sourceMediaType": "application/vnd.ms-excel.template.macroenabled.12",                           "targetMediaType": "text/html"},
+        {"sourceMediaType": "application/vnd.ms-excel.template.macroenabled.12",                           "targetMediaType": "text/plain"},
+        {"sourceMediaType": "application/vnd.ms-excel.template.macroenabled.12",                           "targetMediaType": "application/xhtml+xml"},
+        {"sourceMediaType": "application/vnd.ms-excel.template.macroenabled.12",                           "targetMediaType": "text/xml"},
+
+        {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.template",        "targetMediaType": "text/html"},
+        {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.template",        "targetMediaType": "text/plain"},
+        {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.template",        "targetMediaType": "application/xhtml+xml"},
+        {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.template",        "targetMediaType": "text/xml"},
+
+        {"sourceMediaType": "text/xml",                                                                    "targetMediaType": "text/html"},
```

Comment:
Duplicate entries? Same as lines 237-240?

Reply:
It looks like it, thanks for spotting this.